### PR TITLE
LPS-198800 | Test Fix | Master

### PR DESCRIPTION
**Pull request by rafaellajordao**
# Test Fix for ContentPagesUI#CannotViewPortalVersionInfoOnHomePage

Hi! :)
This pr is corresponding to this ticket: [LPS-198800](https://liferay.atlassian.net/browse/LPS-198800)

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesUI.testcase [hunk 1]
```diff
@@ -117,8 +117,9 @@ definition {
 	@priority = 4
 	test CannotViewPortalVersionInfoOnHomePage {
 		ContentPages.viewFragmentText(
+			contributed = "false",
 			fragmentName = "paragraph",
-			id = "element-text",
+			position = 2,
 			text = "Enjoy using the best DXP on Earth!");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesUI.testcase:123]:
```
		ContentPages.viewFragmentText(
			fragmentName = "paragraph",
			id = "element-text",
			position = 2,
			text = "Enjoy using the best DXP on Earth!");
```

**Review by Tim-Cao — CHANGES_REQUESTED** [review]
Hi @rafaellajordao, would you mind modifying according to comments? Thanks

**Comment by Tim-Cao** [comment]
LGTM!

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**